pages/phases/assessments/interview-scheduling.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: article
+permalink: /hiring-phases/assessments/interview-scheduling/
+section: hiring-phases
+sidenav: hiring-phases
+phase: assessments
+sub-phase-order: 010
+title: Scheduling Interviews with Applicants
+intro: To schedule applicants, work with a team of schedulers or admins who can coordinate SME schedules with applicant availability.
+---
+
+<p class="usa-intro">
+  {{ page.intro }}
+</p>
+
+Before you [schedule training](training.md) for the first round of interviews, divide the SMEs into two groups based on which round of interviews they will conduct. Consider how many applicants passed resume review, as well as any scheduling conflicts you may be aware of at this time. Send SMEs calendar invites for the days or weeks they'll be conducting interviews as soon as possible. These invites should just be a placeholder notification of when interviews will generally take place. To solicit the specific times that SMEs will make available for interviews during this period, send them the <a href="{{site.baseurl}}/toolkit/assessments/email-template-sme-availability/">SME Availability Email</a>.
```

Comment:
should we say here- decide if you're having 2 interview rounds or one first or something? this page assumes 2 rounds which may confuse folks.

Reply:
Done: https://github.com/usds/SME-QA-Website/pull/243/commits/753aa63ac4abfc23c25d3a1de466fa09679dee21